content/05.use-cases-c.md

```diff
@@ -37,6 +37,21 @@ In (Figure @fig:github-diagram), we highlight several elements of good repositor
 GitHub also allows users to describe and store information about software and code dependencies, ensuring that code can run using the same version of software as when it was initially developed.
 This can be achieved through the access of GitHub repositories containing the necessary releases of such software, or through the use of automated Github Actions with specific software versions.
 
+### Automation
+
+<!--*Contributors to this section: PHPB -->
+
+Automation has a strong potential to expand the scale and pace of research in ecology and evolution [see @doi:10.1126/science.abi4692].
+Automation frameworks can be applied from the collection, analysis, scrutiny, to the integration and deployment of data and code in EEB research.
+In this context, small modifications to code and data can be frequently committed and automatically tested (as in continuous integration and continuous deployment practices), allowing for the early detection and correction of errors [@doi:10.1109/MS.2014.58], ultimately mitigating the erosion of confidence in scientific development due to software errors [see @doi:10.12688/f1000research.5930.2].
+In addition to increasing scientific rigor and confidence in ecological software [@doi:10.1890/080141], can help advance in the public dissemination and quality of ecological data [@doi:10.1073/pnas.1710231115].
```

Comment:
needs a little bit more I think.  How does automation help public dissemination of data?  Is there a short phrase that could be added here to clarify?

Reply:
Thanks for catching this, @Aariq ! Rob made edits to this part to clarify. Automation can help the public dissemination of data by automatically deploying reports or figures when data is collected and updated.